test/curator_test.rb

```diff
@@ -0,0 +1,188 @@
+       born: "1923",
+       died: "1971",
+       country: "United States"
+    })
+  end
+
+  def test_it_exists
+    assert_instance_of Curator, @curator
+  end
+
+  def test_initialize
+    assert_equal [], @curator.photographs
+    assert_equal [], @curator.artists
+  end
+
+  def test_it_can_add_a_photograph
+    @curator.add_photograph(@photo_1)
+
+    assert_equal [@photo_1], @curator.photographs
+
+    @curator.add_photograph(@photo_2)
+
+    assert_equal [@photo_1, @photo_2], @curator.photographs
+  end
+
+  def test_it_can_add_an_artist
+    @curator.add_artist(@artist_1)
+
+    assert_equal [@artist_1], @curator.artists
+
+    @curator.add_artist(@artist_2)
+
+    assert_equal [@artist_1, @artist_2], @curator.artists
+  end
+
+  def test_it_can_find_artist_by_id
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+    @curator.add_artist(@artist_3)
+    expected = @curator.find_artist_by_id("1")
+
+    assert_equal expected, @artist_1
+
+    actual = @curator.find_artist_by_id("3")
+
+    assert_equal @artist_3, actual
+  end
+
+  def test_it_can_find_photograph_by_id
+    @curator.add_photograph(@photo_1)
+    @curator.add_photograph(@photo_2)
+    actual = @curator.find_photograph_by_id("2")
+
+    assert_equal @photo_2, actual
+  end
+
+  def test_it_can_find_photographs_by_artist
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+    @curator.add_artist(@artist_3)
+    @curator.add_photograph(@photo_1)
+    @curator.add_photograph(@photo_2)
+    @curator.add_photograph(@photo_3)
+    @curator.add_photograph(@photo_4)
+    expected = [@photo_3, @photo_4]
+    actual = @curator.find_photographs_by_artist(@artist_3)
+
+    assert_equal expected, actual
+  end
+
+  def test_it_can_find_artists_with_multiple_photographs
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+    @curator.add_artist(@artist_3)
+    @curator.add_photograph(@photo_1)
+    @curator.add_photograph(@photo_2)
+    @curator.add_photograph(@photo_3)
+    @curator.add_photograph(@photo_4)
+
+    actual = @curator.artists_with_multiple_photographs
+    expected = [@artist_3]
+
+    assert_equal expected, actual
+  end
+
+  def test_it_can_find_photographs_taken_by_artist_by_country
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+    @curator.add_artist(@artist_3)
+    @curator.add_photograph(@photo_1)
+    @curator.add_photograph(@photo_2)
+    @curator.add_photograph(@photo_3)
+    @curator.add_photograph(@photo_4)
+
+    actual = @curator.photographs_taken_by_artist_from("United States")
+    expected = [@photo_2, @photo_3, @photo_4]
+
+    assert_equal expected, actual
+
+    actual = @curator.photographs_taken_by_artist_from("Argentina")
+    assert_equal [], actual
+  end
+
+  def test_it_can_load_photographs_from_csv
+    @curator.load_photographs('./data/photographs.csv')
+    @curator.photographs.each do |photo|
+      assert_instance_of Photograph, photo
+    end
```

Comment:
this is good, but since you CAN peek into the CSV file it would be good to also check one or two of the attributes of each photo, how many photos you got, etc to make sure that you got all of them and every line in the file (except headers) got turned into objects